jetbrains-rider/src-203/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is there any way to detect that we've changed this other than manually?

Reply:
IDE updates it automatically for me